client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.scss

```
@@ -0,0 +1,70 @@
+.unsaved-changes-indicator__container {
+  display: flex;
+}
+
+.unsaved-changes-indicator {
+  display: inline-flex;
+  gap: 8px;
+  padding: 7px 8px 0;
+  border-radius: 3px;
+  border: 1px solid;
+  font-size: 13px;
+  white-space: nowrap;
+  margin-left: 4px;
+  margin-right: 8px;
+
+  // These use bootstrap colors
+  &--notice {
```

```suggestion
}

.unsaved-changes-indicator--notice {
```
Using the full CSS class means it's easier to search for. If I search for `unsaved-changes-indicator--notice` I want to find the scss declaration for it.